index.md

```diff
@@ -9,29 +9,33 @@ published: true
 
 * All of our user interface designs:
-  * [UI designs for all Tidepool apps on Pixelapse](https://www.pixelapse.com/tidepool)
+  * TODO: UI designs for all Tidepool apps
   * [UI designs and prototype for the Bionic Pancreas on GitHub](https://github.com/tidepool-org/bionicpancreas)
-* A list of Technical References, including:
-  * A Tidepool Platform architectural overview. (TODO)
-  * [Tidepool data model documentation](data-model)
-* Project Management reference, including:
-  * Current and recent sprint boards (on Trello)
-    * [Sprint 34](https://trello.com/b/aN98Vryv/sprint-34-2016-06-30-2016-07-15), [Sprint 33](https://trello.com/b/mwvefWmh/sprint-33-2016-06-02), [Sprint 32](https://trello.com/b/otIDXmuI/sprint-32-2016-05-11-2016-05-30)
-  * [Sprint planning priority documents](https://drive.google.com/a/lookfamily.org/folderview?id=0Bzqw_G5XWp9KY0pZTnFUSVFoUU0&usp=drive_web) (in Google Docs).
-  * Product Backlogs (on Trello)
-    * [Active, force-ranked product backlog](https://trello.com/b/pQtao5gN/active-force-ranked-product-backlog)
-    * "Maybe someday" backlogs for [Blip](https://trello.com/b/iKydvoiJ/backlog-blip), [Uploader](https://trello.com/b/oHy9VXYY/backlog-uploader), [Blip Notes](https://trello.com/b/jjciNmRJ/backlog-blip-notes), [Backend](https://trello.com/b/RUDDN3yq/backlog-backend)
+* All of our technical documentation (a work in progress!):
+  * [General docs](http://developer.tidepool.io/docs/) that don't belong to a specific app/repository, plus:
+    * The Blip [docs](http://developer.tidepool.io/blip/) and [developer guide](http://developer.tidepool.io/blip/docs/StartHere.html)
+    * The Chrome uploader [docs](http://developer.tidepool.io/chrome-uploader/) and [developer guide](http://developer.tidepool.io/chrome-uploader/docs/StartHere.html)
+    * The @tidepool/viz data visualization library [docs](http://developer.tidepool.io/viz/) and [developer guide](http://developer.tidepool.io/viz/docs/StartHere.html)
+  * [Tidepool data model documentation](http://developer.tidepool.io/data-model/)
+* Project Management references, including:
+  * Our current [Work In Progress](https://trello.com/b/sLQWlC52/work-in-progress) Trello board
+  * A few of our recent Done boards in Trello, for [December 2016](https://trello.com/b/iPDyVIa1/done-december-2016), [November 2016](https://trello.com/b/G4LyPDa1/done-november-2016), and [October 2016](https://trello.com/b/U6X3Oi3m/done-october-2016)
+  * The [Active Product Design](https://trello.com/b/EdZQUlp6/active-product-design) board on Trello, which is the immediate source for tasks on the [Work In Progress](https://trello.com/b/sLQWlC52/work-in-progress) board
+  * "Maybe someday" backlogs for [Blip](https://trello.com/b/iKydvoiJ/backlog-blip), [Uploader](https://trello.com/b/oHy9VXYY/backlog-uploader), [Blip Notes](https://trello.com/b/jjciNmRJ/backlog-blip-notes), [Backend](https://trello.com/b/RUDDN3yq/backlog-backend)
```

Comment:
If they're current, I'd leave them. There are quite a few things in there that we could take contributions for :)